### PR TITLE
Basic: adjust for SVN r353268

### PR DESCRIPTION
**Pull request by compnerd**
The versioning script has changed for the preparation for the monorepo
adjustment and to generalise it further.  Update our usage.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @bob-wilson @gottesmm @shajrawi

File: lib/Basic/CMakeLists.txt [hunk 1]
```diff
@@ -30,7 +30,7 @@ macro(find_first_existing_vc_file out_var path)
     )
 endmacro()
 
-set(get_svn_script "${LLVM_MAIN_SRC_DIR}/cmake/modules/GetSVN.cmake")
+set(generate_vcs_version_script "${LLVM_MAIN_SRC_DIR}/cmake/modules/GenerateVersionFromVCS.cmake")
```
Comment (gottesmm) [inline lib/Basic/CMakeLists.txt]: Isn't it VersionFromVCS?

Comment (gottesmm) [reply lib/Basic/CMakeLists.txt]: Nm. I am wrong.

**Comment by gottesmm** [comment]
LGTM.

**Pull request merged**